libjlm/include/jlm/ir/types.hpp

```diff
@@ -358,80 +359,96 @@ create_varargtype()
 	return std::unique_ptr<jive::type>(new varargtype());
 }
 
-/* struct type */
-
-class structtype final : public jive::valuetype {
+/** \brief StructType class
+ *
+ * This class is the equivalent of LLVM's StructType class.
+ */
+class StructType final : public jive::valuetype {
 public:
-	virtual
-	~structtype();
+  ~StructType() override;
 
-	inline
-	structtype(
-		bool packed,
-		const jive::rcddeclaration * dcl)
-	: jive::valuetype()
-	, packed_(packed)
-	, declaration_(dcl)
-	{}
+  StructType(
+    bool isPacked,
+    const jive::rcddeclaration & declaration)
+    : jive::valuetype()
+    , IsPacked_(isPacked)
```

Comment:
Is the _ at the end of the variables names necessary?
They are distinguished by the upper case at the start.

Reply:
Yes. The rule is to have an _ at the end of class attributes. It will otherwise collide with method  ames.